lib/ansible/modules/cloud/amazon/aws_ses_identity.py

```diff
@@ -0,0 +1,447 @@
+    notification_attributes = response['NotificationAttributes']
+    if identity not in notification_attributes:
+        return None
+    return notification_attributes[identity]
+
+
+def update_notification_topic(connection, module, identity, identity_notifications, notification_type):
+    arg_dict = module.params.get(notification_type.lower() + '_notifications')
+    topic_key = notification_type + 'Topic'
+    if topic_key in identity_notifications:
+        current = identity_notifications[topic_key]
+    else:
+        current = None
+
+    if arg_dict is not None and 'topic' in arg_dict:
+        required = arg_dict['topic']
+    else:
+        required = None
+
+    if current != required:
+        call_and_handle_errors(
+            module,
+            connection.set_identity_notification_topic,
+            Identity=identity,
+            NotificationType=notification_type,
+            SnsTopic=required,
+        )
+        return True
+    return False
+
+
+def update_notification_topic_headers(connection, module, identity, identity_notifications, notification_type):
+    arg_dict = module.params.get(notification_type.lower() + '_notifications')
+    header_key = 'HeadersIn' + notification_type + 'NotificationsEnabled'
+    if header_key in identity_notifications:
+        current = identity_notifications[header_key]
+    else:
+        current = False
+
+    if arg_dict is not None and 'include_headers' in arg_dict:
+        required = arg_dict['include_headers']
+    else:
+        required = False
+
+    if current != required:
+        call_and_handle_errors(
+            module,
+            connection.set_identity_headers_in_notifications_enabled,
+            Identity=identity,
+            NotificationType=notification_type,
+            Enabled=required,
+        )
+        return True
+    return False
+
+
+def update_feedback_forwarding(connection, module, identity, identity_notifications):
+    if 'ForwardingEnabled' in identity_notifications:
+        current = identity_notifications['ForwardingEnabled']
+    else:
+        current = False
+
+    required = module.params.get('feedback_forwarding')
+
+    if current != required:
+        call_and_handle_errors(
+            module,
+            connection.set_identity_feedback_forwarding_enabled,
+            Identity=identity,
+            ForwardingEnabled=required,
+        )
+        return True
+    return False
+
+
+def update_identity_notifications(connection, module):
+    identity = module.params.get('identity')
+    changed = False
+    identity_notifications = get_identity_notifications(connection, module, identity)
+
+    for notification_type in ('Bounce', 'Complaint', 'Delivery'):
+        changed |= update_notification_topic(connection, module, identity, identity_notifications, notification_type)
+        changed |= update_notification_topic_headers(connection, module, identity, identity_notifications, notification_type)
```

Comment:
TIL about the `|=` operator. For others reviewing: this will replace changed with True iff changed is not True already and a function evaluates to True. Once True, one of these updated returning False won't switch the changed status.